internal/pkg/policy/policy_output.go

```diff
@@ -73,6 +89,7 @@ func (p *Output) prepareElasticsearch(
 	ctx context.Context,
 	zlog zerolog.Logger,
 	bulker bulk.Bulk,
+	outputBulker bulk.Bulk,
```

Comment:
the `outputBulker` is the same for local ES, and different for remote ES. It is only used to create/update Api keys in the right ES